web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -191,12 +181,8 @@ export default function MyPage() {
   };
 
   const pencilClick = (text: string) => {
```

Comment:
Please modify this method name to something like `handleOnEditButtonClicked` or `onEditButtonClicked`.